spec/broadside/configuration_spec.rb

```diff
@@ -25,18 +25,60 @@
     expect(Broadside.config.valid?).to be false
   end
 
-  context 'ssh' do
+  describe '#ssh_cmd' do
     let(:ip) { '123.123.123.123' }
+    let(:keyfile) { 'path_to_keyfile' }
```

Comment:
also you can move this into the `context 'with configured SSH user and keyfile'` - doesn't need to be at this top level because it's only relevant in that context